src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesEdit.js

```diff
@@ -41,6 +42,7 @@ export default class ExternalDataSourcesEdit extends React.Component {
       onSave,
     } = this.props;
 
+    const isReadonly = value.readonly;
```

Comment:
It actually just gets _more_ confusing sometimes when using variables to hold values like this, because you need to keep more things in memory when scanning the file. Like, I'm always gonna know what `value.readonly` means in the context of this component, but it's easy to forget how and where `isReadonly` was set and defined.

Reply:
Alternatively, if you feel very strongly, leave it but camel-case it: `isReadOnly` 😝

Reply:
I hadn't camel-cased only because the value in the object itself wasn't 😝 but I think I'll remove it... I'm not sure why I did that for something so trivial.